README.md

```diff
@@ -39,6 +39,15 @@
     - run: **python main.py all** to run both the generation and simulation.
     - run: **python main.py [simulate|generate] --help** for more help about each option.
     
+    
+## Installing and running under Linux (tested on Debian 10):
+- sudo apt install swig liblzma-dev libbz2-dev
+    - swig - building parasymoblic matrix needs swig
```

Comment:
can't we pre-build the parasymbolic so that clients don't need swig to run it it?

Reply:
Yes, I can push my build but I'm not sure it will work for every Linux...

Reply:
you can push your CXX file and the clients won't need swig to build